esmvalcore/_config.py

```diff
@@ -61,8 +62,13 @@ def read_config_user_file(config_file, recipe_name):
     cfg['output_dir'] = _normalize_path(cfg['output_dir'])
     cfg['auxiliary_data_dir'] = _normalize_path(cfg['auxiliary_data_dir'])
 
-    cfg['config_developer_file'] = _normalize_path(
-        cfg['config_developer_file'])
+    if cfg['quicklook']['active']:
+        cfg['config_developer_file'] = _normalize_path(
+            cfg['config_developer_file'])
+        cfg['quicklook']['output_dir'] = _normalize_path(
+            cfg['quicklook']['output_dir'])
+        cfg['quicklook']['recipe_dir'] = _normalize_path(
+            cfg['quicklook']['recipe_dir'])
```

Comment:
Can you add

```python
cfg['save_intermediary_cubes'] = False
cfg['remove_preproc_dir'] = False
```
?

I think these options should not be set in the quicklook mode.

Reply:
Done

Reply:
Ahh, sorry, I meant

```python
cfg['remove_preproc_dir'] = True
```

of course.

Reply:
Oh you test me :smile: